build.sbt

```diff
@@ -9,28 +11,24 @@ lazy val commonSettings = Seq(
     case _ => MergeStrategy.first}},
   scalacOptions ++= Seq("-deprecation","-unchecked","-Xsource:2.11"),
   libraryDependencies += "org.scalatest" %% "scalatest" % "3.0.5" % "test",
-  libraryDependencies += "org.json4s" %% "json4s-native" % "3.6.1",
+  libraryDependencies += "org.json4s" %% "json4s-jackson" % "3.6.1",
   libraryDependencies += "org.scala-lang" % "scala-reflect" % scalaVersion.value,
-  libraryDependencies += "edu.berkeley.cs" %% "firrtl-interpreter" % "1.2-SNAPSHOT",
+  //libraryDependencies += "edu.berkeley.cs" %% "firrtl-interpreter" % "1.2-SNAPSHOT",
```

Comment:
Just remove no?

Reply:
Yeah. Do we use it anywhere?

Reply:
I don't and nor does the BOOM team? Others maybe? @colinschmidt @zhemao @azidar

Reply:
Uh at some point barstools used this

Reply:
Alright, should be easy to reintroduce.

Reply:
Will it be used again? @colinschmidt @jwright6323

Reply:
Its currently used to run tests for macro compiler: https://github.com/ucb-bar/barstools/blob/master/build.sbt#L34

Reply:
Do the tests run without this right now if you pull this?

Reply:
I can just fix this when i get back to my machine.